ci-operator/step-registry/openshift/e2e/vsphere/multi-vcenter/csi/openshift-e2e-vsphere-multi-vcenter-csi-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-e2e-vsphere-multi-vcenter-csi
+  steps:
+    pre:
+    - chain: ipi-vsphere-pre-multi-vcenter
+    - ref: storage-conf-csi-vsphere-topology
+    - ref: storage-obj-save
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: storage-obj-check
+    - chain: gather-core-dump
+    - chain: ipi-vsphere-post
+    env:
+      # for storage-conf-csi-vsphere and openshift-e2e-test
+      TEST_CSI_DRIVER_MANIFEST: manifest-vsphere.yaml
+      TEST_OCP_CSI_DRIVER_MANIFEST: ocp-manifest-vsphere.yaml
```

Comment:
where does `ocp-manifest-vsphere.yaml` comes from?

Reply:
There is a [ocp-tests.yaml](https://github.com/openshift/vmware-vsphere-csi-driver-operator/blob/main/test/e2e/ocp-tests.yaml) which is used for LUNStressTest (only this one by now), it is called as OCP_CSI_DRIVER_MANIFEST to distinguish the CSI_DRIVER_MANIFEST. And it comes from:
https://github.com/openshift/release/pull/62944/files#diff-5a19b2712b586f6f394bef0a5b040d5f2d90f7ed730682e92bc191535eb4a77aR10-R13

I just copied from other jobs, hope I didn't make any mistake here :)